src/nvim/buffer.c

```diff
@@ -2605,14 +2605,22 @@ void buflist_list(exarg_T *eap)
       continue;
     }
 
+    const int changed_char = (buf->b_flags & BF_READERR)
+      ? 'x'
+      : (bufIsChanged(buf) ? '+' : ' ');
+    const int ro_char = !MODIFIABLE(buf)
+      ? '-'
+      : (buf->b_p_ro ? '=' : ' ');
```

Comment:
It should be possible to show R or F here, using similar approach as `f_jobpid`. I will send a PR.